certified-connectors/Certopus/apiDefinition.swagger.json

```diff
@@ -0,0 +1,321 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Certopus",
+    "description": "The Connector for Certopus allows users to integrate the Certopus API into their workflows. With this connector, users can easily create and manage digital certificates using the Certopus platform. The connector provides a simple and intuitive interface for interacting with the Certopus API, allowing users to automate tasks such as creating and publishing certificates and more.",
```

Comment:
@VrajGohil Please remove the word "API" from the description as it is not allowed. please follow our requirements [here ](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info)and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).

Reply:
I've remove the word "API" from the description